third_party/python/python.mk

```diff
@@ -2610,9 +2610,9 @@ o/$(MODE)/third_party/python/Lib/test/test_pyexpat.py.runs: \
 		o/$(MODE)/third_party/python/pythontester.com
 	@$(COMPILE) -ACHECK -tT$@ $(PYHARNESSARGS) $< -m test.test_pyexpat $(PYTESTARGS)
 
-o/$(MODE)/third_party/python/Lib/test/test_ioctl.py.runs: \
-		o/$(MODE)/third_party/python/pythontester.com
-	@$(COMPILE) -ACHECK -tT$@ $(PYHARNESSARGS) $< -m test.test_ioctl $(PYTESTARGS)
+#o/$(MODE)/third_party/python/Lib/test/test_ioctl.py.runs: \
```

Comment:
Could you comment out the test code itself and leave a todo? I'm surprised this doesn't break the build, since these tests are still listed under `THIRD_PARTY_PYTHON_CHECKS` via `$(THIRD_PARTY_PYTHON_PYTEST_PYMAINS:%=o/$(MODE)/%.runs)`

Reply:
done